rule/rule.go

```diff
@@ -315,6 +315,17 @@ func (f *File) Sync() {
 	for r, w = 0, 0; r < len(f.Rules); r++ {
 		s := f.Rules[r]
 		s.sync()
+
+		// the package() rule is special in that it is REQUIRED to occur
+		// after all load()s and before any other rules. Therefore we
+		// manually update the index of any Rule{kind:"package"} we
+		// find to the index of the last load(). This is safe because
+		// "statements at the same index will occur in order" and rules
+		// always occur after load()s in the call to updateStmt below.
+		if s.Kind() == "package" {
+			s.index = len(f.Loads)
```

Comment:
The index of last `load` is `len(f.Loads) - 1`, right? So this will give `package` the same index of the first rule statement.

Also, `index` of some statements are rewritten in `updateStmt`. Will the index of `package` also be rewritten?